src/BlockTypes/MiniCart.php

```diff
@@ -221,7 +233,7 @@ protected function get_markup() {
 		}
```

Comment:
Heads-up that you will also need to update the button that is shown when the Mini Cart is disabled. :point_up: